src/core/Alert/Alert/Alert.baseStyles.ts

```diff
@@ -5,35 +5,31 @@ import { baseAlertBaseStyles } from '../BaseAlert/BaseAlert.baseStyles';
 
 export const baseStyles = (theme: SuomifiTheme) => css`
   ${baseAlertBaseStyles(theme)}
-
+  padding-bottom: 7px;
```

Comment:
What is this for? It just seems to add padding that isn't there in the designs or in the previous version

Reply:
This was there, because there was need to add overflow:hidden to parent. Focus ring was then cut. Padding-bottom made that visible, Later overflow:hidden was removed, but this padding, was forgot to be removed. I removed padding.